frontend/src/components/navbar.svelte

```diff
@@ -25,12 +34,33 @@
       <a class="btn btn-ghost btn-sm rounded-btn" href="/about"> About </a>
       <a class="btn btn-ghost btn-sm rounded-btn" href="/faq"> FAQ </a>
     </div>
+    <div class="dropdown dropdown-end">
+      <div tabindex="0" class="btn btn-ghost btn-sm rounded-btn text-xl">🎨</div>
+      <ul
+        tabindex="0"
+        class="p-2 shadow menu dropdown-content bg-base-100 rounded-box w-52"
+      >
+        {#each themes as theme}
+          <li
+            data-set-theme={theme.split(" ")[1]}
+            data-act-class="ACTIVECLASS"
+            on:click={() => ($chosenTheme = theme.split(" ")[1])}
+          >
+            {#if $chosenTheme == theme.split(" ")[1]}
```

Comment:
Can we refactor this a bit? We should probably use an array of objects and not use splits

```js
const themes = ["💎 luxury", "🌚 dark", "🌲 forest", "🎃 halloween", "🌆 synthwave"]
```
to
```js
const themes = [{icon: "💎", value: "luxury"}]
```